### DISCUSSION TITLE
String Currency Symbol Parsing ($)

### DISCUSSION THREAD

**Discussion by nickmagruder** [carbon-design-system/carbon#12070] [Help] [answered]
Does Carbon (V10) do any currency character parsing, and are there ways to configure it? I wasn't able to find anything in the docs. My app has a component where a dollar-amount string is passed to  a <p> element in React, and for some reason it is converting the $ into ¤ when the page renders on the deployed application. A code example of where this is happening is below, where the formattedValue variable is a string of "$100.00" but renders as ¤100.00.

Any suggestions greatly appreciated!

```
return (
                                <><Column style={{ flex: 1, marginLeft: index === 0 ? "0px": "10px" }}>
                                <p style={{ color: "#727A7D", minHeight: "45px", lineHeight: item.title.length >= 28 ? "16px" : "1.5em" }}>{item.title}</p>
                                <p style={{ marginTop: "8px", fontSize: "30px", color }}>{formattedValue}</p>
                                </Column>
                                </>
                            );
```

**Comment by jnm2377** [comment]
Hey @nickmagruder, we don't do any sort of character parsing that would change this. Would you be able to recreate your issue in a sandbox so we can see what's going on? What version of Carbon are you on, and has this always been an issue or a recent one (trying to determine if it's tied to any version bugs)?

**Reply by nickmagruder** [reply]
Hello! I was just able to solve it. Turns out it was a locale setting with our API that was actually sending the ¤ symbol to the front end instead of the $. Thanks for your help though!

**Comment by nickmagruder** [comment, marked as answer]
Wasn't related to Carbon, but was a locale setting in our API.
